updatecli/updatecli.d/azcopy.yml

```
@@ -0,0 +1,72 @@
+---
+name: Bump `azcopy` version
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  lastReleaseVersion:
+    kind: shell
+    name: Get the latest `azcopy` (full) version
+    spec:
+      command: bash -c 'basename "$(dirname "$(curl https://aka.ms/downloadazcopy-v10-linux --write-out "%{redirect_url}" --output /dev/null --silent --fail --show-error)" )"'
+    transformers:
+      - trimprefix: 'release-'
+
+conditions:
+  testDockerfileArgAzCliVersion:
+    name: "Does the Dockerfile have an ARG instruction which key is AZCOPY_VERSION?"
+    kind: dockerfile
+    disablesourceinput: true
+    spec:
+      file: Dockerfile
+      instruction:
+        keyword: "ARG"
+        matcher: "AZCOPY_VERSION"
+  testCstAzCliVersion:
+    name: "Does the test harness checks for a label io.jenkins-infra.tools.azcopy.version?"
+    kind: yaml
+    disablesourceinput: true
+    spec:
+      file: "cst.yml"
+      key: "metadataTest.labels[6].key"
+      value: io.jenkins-infra.tools.azcopy.version
+
+targets:
+  updateCstVersion:
+    name: "Update the label io.jenkins-infra.tools.azcopy.version in the test harness"
+    sourceid: lastReleaseVersion
+    kind: yaml
+    spec:
+      file: "cst.yml"
+      key: "metadataTest.labels[6].value"
```

```suggestion
      key: "$.metadataTest.labels[6].value"
```

As per the output warning:

```
WARNING: current yaml key is "metadataTest.labels[6].value" and should be updated to "$.metadataTest.labels[6].value"
```